graphql/course/mutations.ts

```diff
@@ -193,6 +220,20 @@ export class MutateCourseResolver {
         await hasAccess(context, 'Course', course);
         await prisma.course.update({ data: { courseState: 'submitted' }, where: { id: courseId } });
         logger.info(`Course (${courseId}) submitted by Student (${context.user.studentId})`);
+
+        const subcourse = await prisma.subcourse.findFirst({
+            where: { courseId: courseId },
+            include: { subcourse_instructors_student: { select: { student: true } } },
+        });
+        if (subcourse && subcourse.subcourse_instructors_student) {
+            subcourse.subcourse_instructors_student.forEach(async (instructor) => {
+                await Notification.actionTaken(userForStudent(instructor.student), 'instructor_course_submitted', {
+                    courseName: course.name,
+                    subcourseId: subcourse.id.toString(),
```

Comment:
Why do we need this?

Reply:
This is the id, replacing a value inside the actionRedirectLink of the related template.